### ISSUE TITLE
Site install said to run successfully not installing site

### ISSUE THREAD

**Issue by hkorik** [hechoendrupal/DrupalConsole#2278]
I am using console version 1.0.0-alpha2 on a Mac OS X El Capitan installing a mysql DB on MAMP. I have run `drupal sn` and downloaded drupal 8.1.1 after that I ran `drupal site:install` and went through the interactive mode. I got a message saying `Your Drupal 8 installation was completed successfully.` 

However I didn't see any DB added to my phpmyadmin and the site didn't load. I got a message "The website encountered an unexpected error. Please try again later." when attempting to access the site in my browser. I ran the `/install.php` script from the web browser and I see it takes me through the installation process once more and this time it installs the site. 

Why would the site not be installing through the terminal with site:install and making me do it through the web browser when in the terminal I got a message saying it installed successfully?

**Comment by jmolivas** [comment]
@hkorik  is this a multi-site?

**Comment by hkorik** [comment]
@jmolivas Nope. It's a Drupal 8 single site installation.

**Comment by hkorik** [comment]
I got it working! Not sure if this is related to helping it fix the issue but I found that it started working after rolling back the version of composer I had installed. running `composer self-update` rollbacked the version I had initially installed to `version 1.0.0-alpha10`. I unfortunately don't know exactly what version it was that I had installed originally that was causing it to not work but I do know after changing that version I no longer had any issues with the site:install command or any other composer command for that matter.
